src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java

```diff
@@ -212,14 +216,20 @@ public boolean containsItems() {
             } else {
                 tankInfos.add(invFluids.getTankInfo(this.side));
             }
+            boolean hasTank = false;
             for (FluidTankInfo[] tankInfoArray : tankInfos) {
                 for (FluidTankInfo tank : tankInfoArray) {
+                    hasTank = true;
                     FluidStack fluid = tank.fluid;
                     if (fluid != null && fluid.amount > 0) {
                         return true;
                     }
                 }
             }
+            if (!hasTank && invItems == null) {
```

Comment:
Shouldn't this be `||`?

Reply:
only when both fluid and item inventory aren't existed it can be considered as full